extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -146,6 +176,29 @@ def get(self, objectId: str = None):
                 application/json:
                   schema: Error
         """
+        single_user_group = (
+            DBSession().query(Group).filter(Group.name == self.current_user.username).first()
+        )
+        # grab streams accessible to user:
+        if single_user_group is not None:
+            streams = (
+                DBSession()
+                .query(Stream)
+                .join(GroupStream)
+                .filter(GroupStream.group_id == single_user_group.id)
+                .all()
+            )
+        else:
+            streams = []
+
+        # allow access to public data only by default
+        selector = {1}
```

Comment:
What if the public group != 1? (Hardcording)

Reply:
This refers to `candidate.program_id` in ZTF alerts.